third_party/cryptoc/deps.bzl

```diff
@@ -0,0 +1,14 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
+
+def cryptoc_deps():
+    new_git_repository(
+        name = "cryptoc",
+        build_file = Label("//third_party/cryptoc:BUILD.cryptoc.bazel"),
+        remote = "https://chromium.googlesource.com/chromiumos/third_party/cryptoc/",
+        commit = "e05bfa91102dd5137b4027b4f3405e041ffe2c32",
```

Comment:
Is the SHA-1 hash the only thing we use to check the integrity of the code we are pulling in?

Reply:
As we make things more reproducible we'll want to set sha256 args so they can be checked.

Reply:
https://github.com/lowRISC/opentitan/issues/11792

Reply:
I assumed we had been talking about the commit ID here which is a SHA-1 we implicitly check when we build. the git repository rule has an optional sha256 arg which should help us harden builds against engineered SHA-1 collisions.

Reply:
Because support for sha256 checking isn't included in a git_repository rules anymore, we'll end up preferring http_archives to help us with our guarantees of reproducibility.